jobs/rocoto/wavepostsbs.sh

```diff
@@ -6,25 +6,23 @@ source "${HOMEgfs}/ush/preamble.sh"
 # Source FV3GFS workflow modules
 #. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
 source "${HOMEgfs}/ush/load_ufswm_modules.sh"
-status=$?
-if [[ ${status} -ne 0 ]]; then
-    exit "${status}"
-fi
+err=$?
+if [[ "${err}" -ne 0 ]]; then exit "${err}"; fi
 
 export job="wavepostsbs"
 
 ###############################################################
 # shellcheck disable=SC2153
 IFS=', ' read -r -a fhr_list <<< "${FHR_LIST}"
 
-export FHR3 jobid
+export FORECAST_HOUR jobid
 for FORECAST_HOUR in "${fhr_list[@]}"; do
-	FHR3=$(printf '%03d' "${FORECAST_HOUR}")
-	jobid="${job}_f${FHR3}.$$"
-	# Execute the JJOB
-	"${HOMEgfs}/jobs/JGLOBAL_WAVE_POST_SBS"
-	status=$?
-	[[ ${status} -ne 0 ]] && exit "${status}"
+  fhr3=$(printf '%03d' "${FORECAST_HOUR}")
+  jobid="${job}_f${fhr3}.$$"
+  # Execute the JJOB
+  "${HOMEgfs}/jobs/JGLOBAL_WAVE_POST_SBS"
+  err=$?
+  if [[ "${err}" -ne 0 ]]; then exit "${err}"; fi
```

Comment:
Multiple lines?

Reply:
I'll fix all these inconsistencies in a bit.  Thanks!